### PR TITLE
Added special printing to FloorDiv so it's printed out with // insead of as a name

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #87328
* __->__ #87263
* #87254
* #87252
* #87161

### CODE REVIEW

File: torch/fx/experimental/symbolic_shapes.py [hunk 1]
```diff
@@ -127,7 +128,7 @@ def clone(self):
         return PySymInt(self.expr, self.shape_env, constant=self.constant)
 
     def _update_expr(self):
-        self._expr = self.shape_env.simplify(self._expr)
+        self._expr = self.shape_env.replace(self._expr)
```
Comment (albanD) [inline torch/fx/experimental/symbolic_shapes.py:131]: Really?

**Review by albanD — COMMENTED** [review]
Printing sounds good. But spurious code?

File: torch/fx/experimental/symbolic_shapes.py (hunk 1 above)
Comment (Chillee) [reply torch/fx/experimental/symbolic_shapes.py:131]: Mmm... yeah. Otherwise sympy "helpfully" does simplifications like

```
(x + 1)/5 => x/5 + 1/5
```

**Review by ezyang — APPROVED** [review]

File: torch/fx/experimental/symbolic_shapes.py (hunk 1 above)
Comment (albanD) [reply torch/fx/experimental/symbolic_shapes.py:131]: But isn't that reverting what the PR just below in the stack is doing partially?

Comment (Chillee) [reply torch/fx/experimental/symbolic_shapes.py:131]: Yeah. Ideally I would just modify it in the PR below, but don't wanna restart CI 🤔

**Pull request closed without merging**